metro.config.js

```diff
@@ -1,31 +1,27 @@
-const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
+const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
 
-const defaultConfig = getDefaultConfig(__dirname);
-const { assetExts, sourceExts } = defaultConfig.resolver;
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
 
-const blacklist = require('metro-config/src/defaults/exclusionList');
+const blacklist = require('metro-config/src/defaults/exclusionList')
```

Comment:
You may need ; at the end.

Reply:
; is already there in the file but not sure why its showing in the comment. I have updated again.